src/locales/fr/WCAG.json

```
@@ -0,0 +1,729 @@
+                    "TITLE": "Contrôlée par l’agent utilisateur"
+                },
+                "4": {
+                    "DESCRIPTION": "Une présentation spécifique de la cible est essentielle pour transmettre l’information.",
+                    "TITLE": "Essentielle"
+                }
+            }
+        },
+        "2.5.6": {
+            "TITLE": "Modalités d’entrées concurrentes",
+            "DESCRIPTION": "Le contenu Web ne restreint pas l’utilisation des modalités d’entrée disponibles sur une plateforme, sauf si la restriction est essentielle, nécessaire pour garantir la sécurité du contenu ou nécessaire pour respecter les paramètres de l’utilisateur.",
+            "DETAILS": {}
+        },
+        "3.1.1": {
+            "TITLE": " Langue de la page",
+            "DESCRIPTION": "La langue par défaut de chaque page Web peut être déterminée par un programme informatique.",
+            "DETAILS": {}
+        },
+        "3.1.2": {
+            "TITLE": "Langue d’un passage",
+            "DESCRIPTION": "La langue de chaque passage ou expression du contenu peut être déterminée par un programme informatique sauf pour un nom propre, pour un terme technique, pour un mot dont la langue est indéterminée ou pour un mot ou une expression faisant partie du langage courant de la langue utilisée dans le contexte immédiat.",
+            "DETAILS": {}
+        },
+        "3.1.3": {
+            "TITLE": "Mots rares",
+            "DESCRIPTION": "Un mécanisme est disponible pour identifier la définition spécifique des mots ou expressions utilisés de manière inhabituelle ou de façon limitée, y compris les expressions idiomatiques et le jargon.",
+            "DETAILS": {}
+        },
+        "3.1.4": {
+            "TITLE": " Abréviations",
+            "DESCRIPTION": "Un mécanisme est disponible pour identifier la forme complète ou la signification d’une abréviation.",
+            "DETAILS": {}
+        },
+        "3.1.5": {
+            "TITLE": "Niveau de lecture",
+            "DESCRIPTION": "Lorsqu’un texte nécessite une capacité de lecture plus avancée que le premier cycle de l’enseignement secondaire après la suppression des noms propres et des titres, un contenu additionnel ou une version qui ne requiert pas de capacité de lecture supérieure au premier cycle de l’enseignement secondaire est disponible.",
+            "DETAILS": {}
+        },
+        "3.1.6": {
+            "TITLE": "Prononciation",
+            "DESCRIPTION": "Un mécanisme permet d’identifier la prononciation spécifique des mots dont la signification est ambiguë dans le contexte si leur prononciation n’est pas connue.",
+            "DETAILS": {}
+        },
+        "3.2.1": {
+            "TITLE": "Au focus",
+            "DESCRIPTION": "Quand un composant d’interface utilisateur reçoit le focus, il ne doit pas provoquer de changement de contexte.",
```

```suggestion
            "DESCRIPTION": "Quand un composant d’interface utilisateur reçoit le focus, il ne doit pas initier de changement de contexte.",
```

See https://www.w3.org/Translations/WCAG21-fr/#on-focus
Please use the exact wording of the Authorized Translation

---

Là aussi, un anglicisme. "Initier" en français veut dire "faire découvrir", par "commencer". Je propose de maintenir "provoquer" ou un mot semblable.

---

See https://github.com/w3c/wai-wcag-em-report-tool/pull/127#discussion_r1402411577